tests/api/cluster/test_app.py

```diff
@@ -2,35 +2,21 @@
 
 import pytest
 
-from controlpanel.api.cluster import BASE_ROLE_POLICY, App
-from controlpanel.api import models
-
-from django.conf import settings
+from controlpanel.api import cluster, models
 
 
 @pytest.fixture
-def app_model():
+def app():
```

Comment:
reason why I called this `app_model` was as we're in the context of the `cluster` module which also contains an `App` class and I thought it may lead to confusion. But no need to change, let's see how it goes 👍